examples/hosts/app-integration/external-data/src/model-interface/index.ts

```diff
@@ -30,25 +30,35 @@ export interface IAppModel extends IEventProvider<IAppModelEvents> {
      * A task tracker list.
      */
     readonly taskList: ITaskList;
-
-    /**
-    * Send custom signals to the server which will cause the server to respond
-    * with the (currently experimental) RuntimeMessage Signal to communicate
-    * an external data change and and possibly the changed data as well
-    */
-    readonly debugSendCustomSignal: () => void;
 }
 
 /**
  * Events emitted by {@link ITask}.
  */
 export interface ITaskEvents extends IEvent {
     /**
-     * Emitted when the name or priority have changed respectively.
+     * Emitted when the name has changed.
+     */
+    (event: "nameChanged", listener: () => void);
+    /**
+     * Emitted when the priority has changed.
+     */
+    (event: "priorityChanged", listener: () => void);
+    /**
+     * Emitted when the externalName has changed.
      */
-    (event: "nameChanged" | "priorityChanged", listener: () => void);
+    (event: "externalNameChanged", listener: () => void);
+
+    /**
+     * Emitted when extneralPriority has changed.
+     */
+    (event: "externalPriorityChanged", listener: () => void);
 }
 
+export const DEFAULT_DIFF_PRIORITY: number = Number.POSITIVE_INFINITY;
```

Comment:
Recommend keeping the interface directory as pure interfaces, so moving any consts out to where the implementation will use them.